modules/social_features/social_topic/config/optional/language.content_settings.node.topic.yml

```diff
@@ -0,0 +1,15 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.topic
+  module:
+    - social_content_translation
+third_party_settings:
+  content_translation:
+    enabled: false
```

Comment:
shouldnt this be enabled in the case of social_content_translation being enabled? 🤔

Reply:
> Not sure if we can just remove function `social_content_translation_disable_field_translations`, even if it is placed in the install file, in theory, it might be used by other developers in their code. So maybe we should keep it for BC, but mark as deprecated?

@ribel Revert and marked the function as deprecated

Reply:
@ronaldtebrake `enabling/disablind` this config handles by translations config overrides (compare [here](https://github.com/goalgorilla/open_social/blob/10.3.x/modules/social_features/social_landing_page/src/ContentTranslationDefaultsConfigOverride.php), or [here](https://github.com/goalgorilla/open_social/blob/10.3.x/modules/social_features/social_event/src/ContentTranslationDefaultsConfigOverride.php), or [here](https://github.com/goalgorilla/open_social/blob/10.3.x/modules/social_features/social_group/modules/social_group_flexible_group/src/ContentTranslationDefaultsConfigOverride.php)).
By default, this setting should be disabled and will be enabled when we enable the module `social_content_translation`